src/data/resolver/user.resolver.ts

```diff
@@ -16,7 +17,12 @@ export class UserResolver {
   @Mutation((returns) => UserEntity)
   async createUser(@Arg('UserEntity') userInput: UserInputType): Promise<UserEntity> {
     const user = this.userRepository.create({ ...userInput });
-
+    try {
+      user.password = bcrypt.hashSync(user.password, 10);
+    } catch (error) {
+      console.error(error);
+      throw new Error('Failed to hash given password');
```

Comment:
Do jeito que o coletor de erros padrão do GraphQL funciona, eu acho que essa mensagem vai acabar parando no retorno do GraphQL. Mas o assunto tratamento de erros está logo ali.

Reply:
Fica até o questionamento: será que `Failed to hash given password` é uma boa mensagem para se retornar para o usuário da API? Será que ele realmente precisa saber que a gente tentou fazer o hash e algo deu errado? Mas como dito, o tratamento de erros já já vai ser abordado hehe